guides/browser-support.md

```diff
@@ -0,0 +1,171 @@
+# Browser support
+
+> A friendly guide for building cross-browser web components
+
+## Why support?
+
+The promise of the web is to allow anyone to [access content using any browser
+on any device](https://hacks.mozilla.org/2016/07/make-the-web-work-for-everyone/).
+By building cross-browser compatible web components we ensure that our content,
+tools, and resources remain available for all that seek them, whether they be
+the American public or Bureau staff. We seek to esnure this open accessibility
+by designing and coding for unknown client environments, and continually
+testing our products in as many environments as is practical.
+
+## Tips for successfully building cross-browser components
+
+Cross-browser support doesn't come from a single place, it takes a combination
+of tools and best practices. We've outlined a few high-level topics here, but
+don't let that stop you from reaching further into your toolset. There's really
+no such thing as too much support.
+
+### User centered design
+
+We build features for our users, not to stoke our own egos. When looking at a
```

Comment:
> We build features for our users, not to stoke our own egos.

Maybe be a little more explicit by what you mean here. "We build features for our users, not based on our own technology interests or whims."

Reply:
Thank you, I struggled with what I was trying to say and rewrote this multiple times, interests and whims is far more accurate.